python/ray/experimental/state.py

```diff
@@ -558,19 +573,35 @@ def micros_rel(ts):
             delta_info["function_name"] = info["function_name"]
             delta_info["worker_id"] = info["worker_id"]
             worker = workers[info["worker_id"]]
+            task_t_info = task_table[task_id]
+            task_spec = task_table[task_id]["TaskSpec"]
+            task_spec["Args"] = [oid.hex() if type(oid) is not int else oid
+                                 for oid in task_t_info["TaskSpec"]["Args"]]
+            task_spec["ReturnObjectIDs"] = [oid.hex() for oid in
+                                            (task_t_info["TaskSpec"]
+                                             ["ReturnObjectIDs"])]
+            task_spec["LocalSchedulerID"] = task_t_info["LocalSchedulerID"]
+
+            total_info = dict(delta_info, **task_spec)
```

Comment:
Right, but we're creating two dictionaries, `delta_info` and `total_info` and it seems like `delta_info` isn't being used for anything other than to construct `total_info`, so maybe we don't need `delta_info`.

E.g., couldn't we start by creating `total_info = copy.copy(task_spec)` and then add the fields to `total_info` instead of to `delta_info`. Would that work?

And maybe also add a comment indicating that `total_info` is the information that is displayed when you click on a task in the timeline.